packages/data-explorer-ui/src/components/common/EllipsisContent/ellipsisContent.tsx

```diff
@@ -18,7 +21,8 @@ export const EllipsisContent = ({
   maxLineCount,
 }: OverviewDescriptionProps): JSX.Element => {
   const ellipsisRef = useRef<HTMLDivElement>(null);
-  const { height } = useResizeObserver(ellipsisRef) || {};
+  const { height } =
+    useResizeObserver(ellipsisRef, getBorderBoxSizeHeight) || {};
```

Comment:
Updates to the `useResizeObserver` include the ability to set state on the observed element's content rect - on values that we are only interested in.